ui/admin/config/environment.js

```diff
@@ -131,11 +131,7 @@ module.exports = function (environment) {
     if (API_HOST) ENV.contentSecurityPolicy['connect-src'].push(API_HOST);
 
     // Enable features in development
-    ENV.featureFlags['primary-auth-method'] = true;
-    ENV.featureFlags['oidc'] = true;
-    ENV.featureFlags['oidc-crud'] = true;
-    ENV.featureFlags['credential-store'] = true;
-    //ENV.featureFlags['oidc-account-crud'] = true;
```

Comment:
Deleted feature enablement in development specifically as these are enabled by default now.

Reply:
Nice cleanup 😉